File: 백준/Bronze/8958. OX퀴즈/OX퀴즈.py
T = int(input())

for _ in range(T):
    ox_string = input()
    total_score = 0
    current_score = 0
    
    for char in ox_string:
        if char == 'O':
            current_score += 1
            total_score += current_score
        else:
            current_score = 0
            
    print(total_score)

File: 백준/Bronze/8958. OX퀴즈/OX퀴즈.js
const lines = require('fs').readFileSync(0).toString().trim().split('\n');
const N = Number(lines[0]);

for (let i = 1; i <= N; i++) {
    const oxString = lines[i];
    let totalScore = 0;
    let currentScore = 0;
    
    for (let char of oxString) {
        if (char === 'O') {
            currentScore += 1;
            totalScore += currentScore;
        } else {
            currentScore = 0;
        }
    }
    
    console.log(totalScore);
}